src/operation/operation_model_interface.jl

```diff
@@ -145,6 +145,11 @@ function build_impl!(model::OperationModel)
 end
 
 function build_if_not_already_built!(model; kwargs...)
```

Comment:
@daniel-thom here is where we need to fix the flow to handle the initialization solver set-up.